src/pages/index.js

```diff
@@ -189,6 +189,7 @@ const api = new Api({
   },
 }); // log the error to the console
 
```

Comment:
```js

  handleDeleteButton() {
    this.deleteBtnModal = document.querySelector("#modal-delete");
    this.deleteBtnModal.classList.add(".modal_opened");
  }
```


please, delete it from `Api`


you can't do that in this class.


Any modal should be created via a special **class** in `index.js`